README.md

```diff
@@ -41,23 +41,42 @@ This project is currently in the early stages of development! It will definitely
   GITHUB_ID=<your-github-client-id>
   GITHUB_SECRET=<your-github-client-secret>
   ```
-  
-  Or, if you don't want to do that, just ask me to send you mine. Refer to [this gist](https://gist.github.com/catarak/70c9301f0fd1ac2d6b58de03f61997e3) for creating an S3 bucket for testing, or if you don't want to do that, I will send you my AWS credentials. 
+
+  Or, if you don't want to do that, just ask me to send you mine. Refer to [this gist](https://gist.github.com/catarak/70c9301f0fd1ac2d6b58de03f61997e3) for creating an S3 bucket for testing, or if you don't want to do that, I will send you my AWS credentials.
 5. `$ npm run build`
 6. `$ npm run start:prod`
 
 ###For Production Setup with PM2
 1. `$ npm install -g pm2`
 2. `$ pm2 start ecosystem.json`
 
+##Accessibility Guidelines
+
+This p5 IDE is developed keeping accessibility in mind. The following guidelines will help to ensure that we keep this a priority as the IDE continues to develop.
+
+**Code Structure**
+
+* Screenreaders (assistive tech for vision loss) know what they are focused on because of the html tags. Use specific tags - nav, ul, li, section etc.. the 'div' tag is probably the least screen reader friendly tag. - [semantic html example](http://html5doctor.com/element-index/#body)
+* All buttons/links/windows need to be accessible by the keyboard ( By tabbing, pressing space etc.)
+* In cases where tags that are not screenreader friendly, we can take advantage of [tabIndex](http://webaim.org/techniques/keyboard/tabindex). Using tabIndex ensures that elements are accessible is a user is using a keyboard as their method of interacting with a site (blind users, users with mobility issues, etc) [code example](https://github.com/processing/p5.js-web-editor/blob/master/client/modules/IDE/components/Editor.js#L249)
+* When a opening a new window/pop up window, ensure the keyboard focus also moves to the new window. [code example](https://github.com/processing/p5.js-web-editor/blob/master/client/modules/IDE/components/NewFileForm.js#L10)
+
+**Labeling**
+
+* While creating buttons icons or images (not a native button or no text, for example) - use [aria-labels](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute). [code example](https://github.com/processing/p5.js-web-editor/blob/master/client/modules/IDE/components/Toolbar.js#L60)
```

Comment:
When creating button icons, images, or something without text (this does not include an HTML5 `<button>`), use [aria-labels](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute). [code example](https://github.com/processing/p5.js-web-editor/blob/master/client/modules/IDE/components/Toolbar.js#L60)